tools/accuracy_checker/accuracy_checker/annotation_converters/squad.py

```diff
@@ -99,7 +99,7 @@ def __init__(
         self.char_to_word_offset = char_to_word_offset
 
         # Start and end positions only has a value during evaluation.
-        if start_position_character is not None and not is_impossible:
+        if start_position_character is not None and not is_impossible and answer_text is not None:
```

Comment:
in which situation answer_text is not None?

Reply:
please check that your changes does not break bert models evaluation

Reply:
bert models work well with these changes